PIM/Integration/03_ManageAttributeGroups.md

```diff
@@ -0,0 +1,263 @@
+
+5. Enter a name for the attribute sub-group in the *Name* field.
+
+6. Enter a key for the attribute sub-group in the *Key* field.
+
+7. Click the [SAVE] button.   
+  The attribute sub-group has been created. The *Add Element* window is closed. The new attribute sub-group is displayed in the attribute sub-group column.
+
+  ![Attribute sub-group created](../../Assets/Screenshots/PIM/Settings/AttributeGroups/AttributeSubGroupCreated.png "[Attribute sub-group created]")
+
+  > [Info] If you want to change the order of the attribute sub-groups in the list, see [Change the order of attribute groups](#change-the-order-of-attribute-groups).
+
+8. Click the [SAVE] button in the upper right corner.  
+  The new attribute sub-group has been saved. The *Edit Attribute Groups* view is closed.
+
+
+
+### Change the order of attribute groups
+
+The order of attribute groups in the *ATTRIBUTE GROUPS* tab equals to the order of attributes groups in the product view. You can change the order to display an attribute group in a different place when creating or editing a product.   
+The procedure to change the order of attribute groups and sub-groups is identical. Just select the attribute sub-group instead of the group and follow the steps described below.
+
+#### Prerequisites
+
+At least two attribute groups have been created, see [Create an attribute group](#create-an-attribute-group).
+
+#### Procedure
+*PIM > Settings > Tab ATTRIBUTE GROUPS*
+
+![Attribute groups](../../Assets/Screenshots/PIM/Settings/AttributeGroups/AttributeGroups.png "[Attribute groups]")
+
+1. Click the ![Edit](../../Assets/Icons/Edit02.png "[Edit]") (Edit) button to the right of the *Attribute Groups* section in the left column.   
+  The *Edit Attribute Groups* view is displayed.
+
+  ![Edit attribute groups](../../Assets/Screenshots/PIM/Settings/AttributeGroups/EditAttributeGroups.png "[Edit attribute groups]")
+
+2. Click and hold the ![Sort](../../Assets/Icons/Sort01.png "[Sort]") (Sort) button to the left of the attribute group you want to move to another position in the list.
+
+3. By using drag and drop, move the selected attribute group to the desired position in the list.
+
+4. Click the [SAVE] button in the upper right corner.  
+  The new attribute group order has been saved. The *Edit Attribute Groups* view is closed.
+
+
+
+### Delete an attribute group
+
+You can delete an attribute group if it is no longer in use. Note, that you just delete the attribute group but not the attributes within a group. All active attributes that are no longer assigned to an attribute group are automatically assigned to a new attribute group with the name *Unassigned Group* which is only displayed in the product view. If you delete an attribute group with sub-groups, also the attribute sub-groups are deleted. The procedure to delete attribute groups and sub-groups is identical. Just select the attribute sub-group instead of the group and follow the steps described below.
```

Comment:
```suggestion
You can delete an attribute group if it is no longer in use. Note that you just delete the attribute group but not the attributes within a group. All active attributes that are no longer assigned to an attribute group are automatically assigned to a new attribute group with the name *Unassigned Group*, which is only displayed in the product view. If you delete an attribute group with sub-groups, also the attribute sub-groups are deleted. The procedure to delete attribute groups and sub-groups is identical. Just select the attribute sub-group instead of the group and follow the steps described below.
```

Reply:
"only displayed in the product view"

- Woauf bezieht sich "product view"?